src/rp2_common/hardware_i2c/include/hardware/i2c.h

```diff
@@ -48,6 +48,10 @@ extern "C" {
 
 typedef struct i2c_inst i2c_inst_t;
 
+// PICO_CONFIG: PICO_DEFAULT_I2C, Define the default I2C for a board, default=undefined, group=hardware_i2c
+// PICO_CONFIG: PICO_DEFAULT_I2C_SDA_PIN, Define the default I2C TX SDA, min=0, max=29, default=undefined, group=hardware_i2c
```

Comment:
IIRC SDA is bidirectional, so I suspect the "I2C TX SDA" here should have been "I2C SDA PIN" ?